benchmark/helper.jl

```diff
@@ -8,7 +8,7 @@ function geo_types_for_spatial_dim(spatial_dim)
     spatial_dim == 3 && return [Tetrahedron, Hexahedron] # Quadratic* not yet functional in 3D. 3D triangle missing. Embedded also missing.
 end
 
-default_refshape(t::Type{C}) where {C <: Ferrite.AbstractCell} = typeof(Ferrite.default_interpolation(t)).parameters[2]
```

Comment:
We can eliminate this hack with the new interface (because the reshape is now part of the cell definition).

Reply:
I was thinking we can do `geo_type.super.parameters[1]` or dispatch `getrefshape` for cell types
Did `geo_type.super.parameters[1]` for now not to add new functionality if it's only used here, what do you think?